### PR TITLE
ODP-1 - Explore data

### PR DESCRIPTION
**Pull request by demenech**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: deployment/frontend/src/pages/search.tsx [hunk 1]
```diff
@@ -5,17 +5,67 @@ import Recent from '@/components/Recent'
 import Footer from '@/components/_shared/Footer'
 import { useState } from 'react'
 import { Filter } from '@/interfaces/search.interface'
+import { api } from '@/utils/api'
+import Spinner from '@/components/_shared/Spinner'
+import { ErrorAlert } from '@/components/_shared/Alerts'
+import RedirectedSearchInput from '@/components/search/RedirectedSearchInput'
 
 export default function test() {
```
Comment (luccasmmg) [inline deployment/frontend/src/pages/search.tsx:13]: is this function name correct?

File: deployment/frontend/src/components/home/HighlightsCarousel.tsx [hunk 2]
```diff
@@ -1,123 +1,128 @@
...
 export interface TopicProps {
-  title: string;
-  numOfDatasets: number;
-  img: string;
+    title: string
+    numOfDatasets: number
+    img: string
 }
 
 const highlights = [
```
Comment (luccasmmg) [inline deployment/frontend/src/components/home/HighlightsCarousel.tsx:17]: are you still hard coding these values?

**Review by luccasmmg — APPROVED** [review]
lgtm

File: deployment/frontend/src/pages/search.tsx (hunk 1 above)
Comment (demenech) [reply deployment/frontend/src/pages/search.tsx:13]: Yes, it's a "redirected search"

File: deployment/frontend/src/components/home/HighlightsCarousel.tsx (hunk 2 above)
Comment (demenech) [reply deployment/frontend/src/components/home/HighlightsCarousel.tsx:17]: This component just uses another component that I had to change, it's part of a page that's not wired up

**Pull request merged**